tests/functional/api/test_moderation.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+
+# String type for request/response headers and metadata in WSGI.
+#
+# Per PEP-3333, this is intentionally `str` under both Python 2 and 3, even
+# though it has different meanings.
+#
+# See https://www.python.org/dev/peps/pep-3333/#a-note-on-string-types
+native_str = str
+
+
+@pytest.mark.functional
+class TestPutHide(object):
+
+    def test_it_returns_http_204_for_group_creator(self,
+                                                   app,
+                                                   group_annotation,
+                                                   user_with_token):
+        user, token = user_with_token
+        headers = {'Authorization': str('Bearer {}'.format(token.value))}
+
+        res = app.put('/api/annotations/{id}/hide'.format(id=group_annotation.id),
+                                                          headers=headers)
+
+        # The creator of a group has moderation rights over the annotations in that group
+        assert res.status_code == 204
```

Comment:
It doesn't look like you've made any changes that affect the status code here, but should this be 201 ("Created")?

Reply:
That's a great question and one that should be asked across the entire API.